ledger/ledger-consensual/src/main/kotlin/net/corda/v5/ledger/consensual/transaction/ConsensualSignedTransactionChecker.kt

```diff
@@ -18,7 +18,7 @@ import java.io.Serializable
  *
  * @see ConsensualLedgerService.receiveFinality
  */
-fun interface ConsensualSignedTransactionVerifier : Serializable {
+fun interface ConsensualSignedTransactionChecker : Serializable {
```

Comment:
What is the rationale for calling this `...Checker` instead of `...Verifier`, when the function it provides is `verify` and not `check`?

Reply:
It isn't verify if we have and call state verify too. It is a check of other context details. We could delete verify on states, but then this makes it more divergent from UTXO

Reply:
I'm not a fan of renaming it to "checker" since it just sounds odd. That being said, I also understand that its not a strong reason not to do it.

This interface is used in the same way as the callback in `SignTransactionFlow` in C4. It is supposed to be separate from state/contract verification.

Reply:
Acceptor?

Reply:
I'm also not a fan of `checker`, but I can't think of another good name to disambiguate from `verifier`/`verify`.

Validator?